examples/minimal-logs/test-bootstrap-scripts/cloudwatch-install.sh

```diff
@@ -0,0 +1,122 @@
+                         \"log_group_name\": \"${log_group}\",
+                         \"log_stream_name\": \"${log_stream}\",
+                         \"timestamp_format\": \"%H: %M: %S%y%b%-d\"
+                     },
+                     {
+                         \"file_path\": \"/home/ubuntu/tamr/logs/unify-all.out.log\",
+                         \"log_group_name\": \"${log_group}\",
+                         \"log_stream_name\": \"${log_stream}\",
+                         \"timestamp_format\": \"%H: %M: %S%y%b%-d\"
+                     },
+                     {
+                         \"file_path\": \"/home/ubuntu/tamr/logs/unify-events.log\",
+                         \"log_group_name\": \"${log_group}\",
+                         \"log_stream_name\": \"${log_stream}\",
+                         \"timestamp_format\": \"%H: %M: %S%y%b%-d\"
+                     },
+                     {
+                         \"file_path\": \"/home/ubuntu/tamr/logs/unify.log\",
+                         \"log_group_name\": \"${log_group}\",
+                         \"log_stream_name\": \"${log_stream}\",
+                         \"timestamp_format\": \"%H: %M: %S%y%b%-d\"
+                     },
+                     {
+                         \"file_path\": \"/home/ubuntu/tamr/logs/auth.log\",
+                         \"log_group_name\": \"${log_group}\",
+                         \"log_stream_name\": \"${log_stream}\",
+                         \"timestamp_format\": \"%H: %M: %S%y%b%-d\"
+                     },
+                     {
+                         \"file_path\": \"/home/ubuntu/tamr/logs/dataset.log\",
+                         \"log_group_name\": \"${log_group}\",
+                         \"log_stream_name\": \"${log_stream}\",
+                         \"timestamp_format\": \"%H: %M: %S%y%b%-d\"
+                     },
+                     {
+                         \"file_path\": \"/home/ubuntu/tamr/logs/recipe.log\",
+                         \"log_group_name\": \"${log_group}\",
+                         \"log_stream_name\": \"${log_stream}\",
+                         \"timestamp_format\": \"%H: %M: %S%y%b%-d\"
+                     },
+                     {
+                         \"file_path\": \"/home/ubuntu/tamr/logs/dedup.log\",
+                         \"log_group_name\": \"${log_group}\",
+                         \"log_stream_name\": \"${log_stream}\",
+                         \"timestamp_format\": \"%H: %M: %S%y%b%-d\"
+                     }
+                 ]
+              }
+          }
+      }
+  } " | tee cloudwatch-config.json
+
+case $os in
+       "\"Amazon Linux\"")
+              echo "********** Mounting Cloudwatch Agent config file to Cloduwatch Agent **********"
+              /opt/aws/amazon-cloudwatch-agent/bin/amazon-cloudwatch-agent-ctl -a fetch-config -m ec2 -s -c file:/opt/aws/amazon-cloudwatch-agent/etc/cloudwatch-config.json
```

Comment:
This command is the same for every OS. Could this be done before the case statement instead?

Reply:
We used the yum package manager for AL2 and rpm in rhel to bypass rhel's subscription management but we can do rpm -i on both of them.

Reply:
I put the command to mount the agent file outside the case statement and I also put together Amazon Linux and RHEL patterns with an OR condition. In addition to this, for the purpose of this example, I hardcoded the `log_streams` instead of passing them as variables. 
Creation of cloudwatch`log_group` and deprecation of `s3_policy_arns` in favor of `additional_policy_arns` will be submitted after merging these changes.